Common/tests/Pollster/TaskHandlerTest.cs

```diff
@@ -876,7 +876,7 @@ public void Dispose()
     {
     }
 
-    public IAsyncPipe<ProcessReply, ProcessRequest>? Pipe { get; private set; }
```

Comment:
This should not be impacted

Reply:
Looks like Resharper decided to remove it during clean up

Reply:
This should not be changed in this PR

Reply:
removed